service/matching/config.go

```diff
@@ -313,6 +315,8 @@ func newTaskQueueConfig(tq *tqid.TaskQueue, config *Config, ns namespace.Name) *
 		GetUserDataLongPollTimeout: config.GetUserDataLongPollTimeout,
 		GetUserDataMinWaitTime:     1 * time.Second,
 		GetUserDataReturnBudget:    returnEmptyTaskTimeBudget,
+		GetUserDataInitialRefresh:  ioTimeout,
+		GetUserDataRefresh:         config.GetUserDataLongPollTimeout, // this is just for testing, use same value as long poll
```

Comment:
the comment made me think this config is used only in tests. Maybe should be more clear. Also, I'm not sure the two configs are inherently connected. Say if we want to change one we don't necessarily want to change the other. So should we just make a separate config for it for sake of not having to think about the coupling?

Reply:
I'll re-word.

They seem connected since in the steady state, GetUserDataLongPollTimeout controls the interval of rpcs from children to parents, and now also the interval of calls from the root to the db.

I can't really think of any reason to change either of them, tbh, so I was trying to avoid adding more configs that will never be used. I can do it though if you think we might change this